.flake8

```diff
@@ -6,11 +6,13 @@ max-line-length = 120
 # E501 is not flexible enough, we're using B950 instead
 ignore =
     E203,E305,E402,E501,E721,E741,F405,F821,F841,F999,W503,W504,C408,E302,W291,E303,
+    # fix these lints in the future
```

Comment:
could you also open an issue to track these growing lists?

Reply:
Would that be really more fruitful than tracking in the flake8 config? I worry the issue would rapidly go out of date or be forgotten.